src/java.base/share/classes/jdk/internal/math/DoubleConsts.java

```diff
@@ -73,12 +73,20 @@ private DoubleConsts() {}
      */
     public static final long    SIGNIF_BIT_MASK = 0x000FFFFFFFFFFFFFL;
 
+    /**
+     * Bit mask to isolate the magnitude bits (combined exponent and
+     * significand fields) of a {@code double}.
+     */
+    public static final int     MAG_BIT_MASK = 0x7FFFFFFF;
```

Comment:
Why is this an int in the DoubleConst class? It now differs also from previous patch, value is different, too.

I wonder why the asserts don't catch this!